src/calendar_duration.rs

```diff
@@ -23,7 +24,7 @@ use crate::{expect, try_opt};
 /// // Encoding 1½ year as a duration in seconds:
 /// let _duration = CalendarDuration::new().seconds(548 * 24 * 60 * 60);
 /// ```
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
```

Comment:
Let's keep this out of the initial commit if you're going to replace it in the second commit, anyway.

Reply:
We have this cfg in lib.rs: `#![deny(missing_debug_implementations)]`, so the initial version can't compile without.

Reply:
Okay, feel free to kill that!